Source/Core/DolphinQt/Debugger/NewBreakpointDialog.h

```diff
@@ -56,4 +63,8 @@ class NewBreakpointDialog : public QDialog
 
   QDialogButtonBox* m_buttons;
   BreakpointWidget* m_parent;
+
+  bool m_edit;
+  TBreakPoint* m_instruction_breakpoint{nullptr};
```

Comment:
Holding pointers here sounds like a recipe for crashes. Can't you pass an optional, a variant, a reference to an overload or getting the right BP based on its address when needed rather than relying on theses pointers?